### PR TITLE
2.1.x add faq entry for supported data types, question asked on stackexchange

### PR DESCRIPTION
**Pull request by iamveritas**
add faq entry for supported data types, question asked on stackexchange.
also correct wrong usage of single quote that messed up a statement, see second commit.
sister PR
https://github.com/EOSIO/welcome/pull/394
https://github.com/EOSIO/welcome/pull/395

### CODE REVIEW

File: docs/96_faq/index.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ You should not call `onblock()` action yourself, but as a blockchain developer y
 
 See [onblock](https://developers.eos.io/manuals/eosio.contracts/latest/action-reference/eosio.system/index/?query=onblock&page=1#onblock)
 
+### What are the supported data types that can be used in an actions parameters?
+
+An action parameter type can be a built-in type or it can be a user defined type.
```
Comment (bobgt) [inline docs/96_faq/index.md]: Can hyphenate `user-defined` just like built-in.

File: docs/96_faq/index.md [hunk 2]
```diff
@@ -108,6 +108,12 @@ You should not call `onblock()` action yourself, but as a blockchain developer y
...
+More details can be found on [Understanding ABI Files](https://developers.eos.io/manuals/eosio.cdt/v1.8/best-practices/abi/understanding-abi-files) documentation chapter and pay special attention to the [Types](https://developers.eos.io/manuals/eosio.cdt/v1.8/best-practices/abi/understanding-abi-files#types) section. The list of supported built-in types can be found [here](https://github.com/EOSIO/eos/blob/release/2.1.x/libraries/chain/abi_serializer.cpp#L85).
+Be aware that EOSIO, although is doing the best it can to support user defined types, if they are too complex it might not be possible to be supported and you will see errors generated; if that is the case you have to simplify the definition of the user defined type.
```
Comment (bobgt) [inline docs/96_faq/index.md]: Can hyphenate `user-defined`

**Review by bobgt — APPROVED** [review]
Approving with minor hyphenation suggestions.

File: docs/40_smart-contract-guides/60_adding-inline-actions.md [hunk 3]
```diff
@@ -80,9 +80,10 @@ The action constructor requires a number of parameters.
 - The action (initialised using `eosio::name` type)
 - The data to pass to the action, a tuple of positionals that correlate to the actions being called.
 
-## The Permission struct
+## The Permission Struct
+
+In this contract the permission should be authorized by the `active` authority of the contract using `get_self()`. As a reminder, to use the `active` authority inline you need for your contract to give active authority to `eosio.code` pseudo-authority (instructions above)
 
-In this contract the permission should be authorized by the `active` authority of the contract using `get_self()`. As a reminder, to use the 'active` authority inline you will need your contract's to give active authority to `eosio.code` pseudo-authority (instructions above)
```
Comment (iamveritas) [inline docs/40_smart-contract-guides/60_adding-inline-actions.md:85]: here 'active` was using single quote and messed up the rest of the statement

**Pull request merged**